token/program-2022/src/pod.rs

```diff
@@ -0,0 +1,99 @@
+//! Rewrites of the base state types represented as Pods
+
+use {
+    crate::{instruction::MAX_SIGNERS, state::PackedSizeOf},
+    bytemuck::{Pod, Zeroable},
+    solana_program::{program_pack::IsInitialized, pubkey::Pubkey},
+    spl_pod::primitives::PodU64,
+};
+
+/// Mint data stored as a Pod type
```

Comment:
Maybe link to the original type doc?
It would also be nice to have a unit test or something that ensures the Pod structs can never diverge from the original type, perhaps by comparing packed versions?

Reply:
Certainly, done!